starwars/src/components/SWData.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState, useEffect} from "react";
+import axios from "axios";
+import SWCard from "./SWCard";
+
+
+
+export default function SWData() {
+    const [characters, setCharacters] = useState([]);
+
+useEffect(() => {
+    axios
+    .get("https://swapi.co/api/people/")
+    .then(response => {
+        console.log(response.data.results);
+        setCharacters(response.data.results);
+    })
+    .catch(error => {
+        console.log("There is an error", error);
+    
+    });
+}, []);
+
+return (
```

Comment:
This will work, but a DRY-er way if you're going to be using most/all of the keys on an object is to just pass the whole thing. In this case, something like `char = {char}`